client/jetpack-cloud/sections/partner-portal/assign-license-form/index.tsx

```diff
@@ -0,0 +1,98 @@
+import { Button, Card } from '@automattic/components';
+import { getQueryArg } from '@wordpress/url';
+import { useTranslate } from 'i18n-calypso';
+import page from 'page';
+import { ReactElement, useCallback, useState } from 'react';
+import { useDispatch } from 'react-redux';
+import FormRadio from 'calypso/components/forms/form-radio';
+import SearchCard from 'calypso/components/search-card';
+import { addQueryArgs } from 'calypso/lib/url';
+import { recordTracksEvent } from 'calypso/state/analytics/actions';
+import { errorNotice, infoNotice } from 'calypso/state/notices/actions';
+import useAssignLicenseMutation from 'calypso/state/partner-portal/licenses/hooks/use-assign-license-mutation';
+import './style.scss';
+
+export default function AssignLicenseForm( { sites }: any ): ReactElement {
+	const translate = useTranslate();
+	const dispatch = useDispatch();
+	const [ filter, setFilter ] = useState( false );
+	const [ selectedSite, setSelectedSite ] = useState( false );
+	const [ isSubmitting, setIsSubmitting ] = useState( false );
+	const licenseKey = getQueryArg( window.location.href, 'key' ) as string;
+	const onSelectSite = ( site: any ) => setSelectedSite( site );
+
+	const siteCards = sites.map( ( site: any ) => {
+		if ( -1 !== site.domain.search( filter ) || false === filter ) {
+			return (
+				<Card key={ site.ID } className="assign-license-form__site-card">
+					<FormRadio
+						className="assign-license-form__site-card-radio"
+						label=""
+						name="site_select"
+						onClick={ () => onSelectSite( site.ID ) }
+					/>
+					{ site.domain }
+				</Card>
```

Comment:
So I added the label text in https://github.com/Automattic/wp-calypso/pull/60903/commits/9035c9ea6329895054bafb6d41f11392d381d966 but the text still is not clickable. I've also added an additional Asana task to work on this, as it could quickly become out of scope for this PR.

Reply:
It's a bit of a pet peeve of mine, so I'm a extra subjective here 😄  I'm a strong advocate for the text to be clickable since that's what people naturally clicks/draws their attention on (_"It's THAT site (read: text) I want to assign it to"_) and an old report I read had it as one of the top bug reports across web development projects because users would report it to not work at all.
The report is close to 10 years ago by now and people have become more used to "quirks" on the internet, so I don't mind it being live for a while considering out target audience is agencies as well.

TL;DR of the report:  "Non devs" would not think to click the radio `O`, so they ended up reporting the functionality to not work at all - not just a bad user experience.